server/api/org/update/controller.js

```diff
@@ -0,0 +1,98 @@
+import Joi from 'joi';
+import Boom from 'boom';
+import compose from 'koa-compose';
+import packJSONRPC from '../../../middleware/packJSONRPC';
+import { validateRequest } from '../../../middleware/validation';
+import {
+  decorateSession,
+  isUserAuthenticated,
+  decorateUserPermissions,
+  findUserPermissionIndex,
+} from '../../../middleware/auth';
+import updateOrg, { checkOrgExists } from './service';
+import { AuthorizationError, OrgNotFoundError } from '../../../constants/errors';
+
+export const validationSchema = {
+  body: {
+    id: Joi.string()
+      .length(24)
+      .hex()
+      .required(),
+    name: Joi.string()
+      .trim()
+      .min(3)
+      .max(100)
+      .optional(),
+    slug: Joi.string()
```

Comment:
Same as above...

```javascript
slug: Joi.string()
  .lowercase()
  .trim()
  .min(3)
  .max(30)
  .regex(/^[A-Za-z0-9\-_]*$/, { name: 'slug' })
  .when('name', {
    is: Joi.exist(),
    then: Joi.optional(),
    otherwise: Joi.required(),
  })
```